src/base_types.ts

```diff
@@ -12,6 +12,7 @@ export interface ContextParams {
   SESSION_STORAGE?: SessionStorage;
   LOG_FILE?: string;
   USER_AGENT_PREFIX?: string;
+  EVENTBRIDGE_ARN: string;
```

Comment:
I'm not sure we want to have this parameter in `Context`, since it's more of a global default than a library config. That said, making this a static setting in `Shopify.Webhooks.Registry` seems less than ideal. I'm torn 😄 

How much of a problem do you think it would be to simply not have a default for this? As far as I can see, apps will usually register webhooks in a single place after OAuth completes so it would be easy to just put the address in a variable and use it for all webhooks.

Reply:
To be honest I wasn't sure either! I agree with the idea of not "polluting" the global context with variables that aren't used globally. And as you say, registering webhooks seems like it would happen in limited situations anyway (I'm no expert). Definitely happy to remove it from context if you think that's sensible.